src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Bulk/submit_async.html.twig

```diff
@@ -0,0 +1,41 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<button id="{{ '%s_grid_bulk_action_%s'|format(grid.id, action.id) }}"
+        class="dropdown-item js-bulk-action-submit-async-btn"
+        type="button"
+        data-form-url="{{ path(action.options.submit_route, action.options.route_params) }}"
+        data-form-method="{{ action.options.submit_method }}"
+        data-confirm-message="{{ action.options.confirm_message }}"
+        data-chunk-size="{{ action.options.chunk_size }}"
+        {% if action.options.modal_options %}
+          data-close-button-label="{{ action.options.modal_options.options.close_button_label|default('Close'|trans({}, 'Admin.Actions')) }}"
+          data-confirm-title="{{ action.options.modal_options.options.title }}"
+          data-confirm-button-class="{{ action.options.modal_options.options.confirm_button_class|default('btn-primary') }}"
+          data-confirm-button-label="{{ action.options.modal_options.options.confirm_button_label|default('Confirm'|trans({}, 'Admin.Actions')) }}"
```

Comment:
I'm thinking maybe we should add some "lazy loading/access" in the `ModalOptions` class, so that we wouldn't need to access the options via the the options field and then we could simply use `action.options.modal_options.confirm_button_class` in the template?